features/docs/writing_support_code/world_namespaces.feature

```diff
@@ -0,0 +1,186 @@
+Feature: World Namespaces
```

Comment:
Oh wait I see you already have the first one below as `A single module is imported without namespace`